### PR TITLE
Updated description of self hosted cluster guides

### PR DESCRIPTION
**Pull request by lucasvaltl**
## Description
- Updated each page that describes a cloud cluster creation guide to be in line with the new intended use of the guides: to create a cluster, not create a cluster AND install Gitpod

## How to test
Spin up workspace, open preview on port 3000, navigate to Docs > Self-Hosted> Cluster Set-Up. Then you can click on each of the guides to see the respective pages.

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1971"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1971"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/cluster-set-up/on-amazon-eks.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ title: Amazon Elastic Kubernetes Service (EKS)
 
 # How to Create a Cluster on Amazon Elastic Kubernetes Service (EKS)
 
-Installation instructions for Gitpod Self-Hosted on Amazon EKS are currently located in the [gitpod-io/gitpod-eks-guide](https://github.com/gitpod-io/gitpod-eks-guide) repository on GitHub. The installation process takes around forty minutes. In the end, the following resources are created:
+Instructions on how to greate an Amazon EKS environment that is ready to install Gitpod are located in the [gitpod-io/gitpod-eks-guide](https://github.com/gitpod-io/gitpod-eks-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
```
Comment (MrSimonEmms) [inline gitpod/docs/self-hosted/latest/cluster-set-up/on-amazon-eks.md:12]:
```suggestion
Instructions on how to create an Amazon EKS environment that is ready to install Gitpod are located in the [gitpod-io/gitpod-eks-guide](https://github.com/gitpod-io/gitpod-eks-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
```

File: gitpod/docs/self-hosted/latest/cluster-set-up/on-gke.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ title: Google Kubernetes Engine (GKE)
 
 # How to Create a Cluster on Google Kubernetes Engine (GKE)
 
-Installation instructions for Gitpod Self-Hosted on Google Kubernetes Engine are currently located in the [gitpod-io/gitpod-gke-guide](https://github.com/gitpod-io/gitpod-gke-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
+Instructions on how to greate an Google GKE environment that is ready to install Gitpod are located in the [gitpod-io/gitpod-gke-guide](https://github.com/gitpod-io/gitpod-gke-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
```
Comment (MrSimonEmms) [inline gitpod/docs/self-hosted/latest/cluster-set-up/on-gke.md:12]:
```suggestion
Instructions on how to create a Google GKE environment that is ready to install Gitpod are located in the [gitpod-io/gitpod-gke-guide](https://github.com/gitpod-io/gitpod-gke-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
```

File: gitpod/docs/self-hosted/latest/cluster-set-up/on-microsoft-aks.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ title: Microsoft Azure Kubernetes Service (AKS)
 
 # How to Create a Cluster on Microsoft Azure Kubernetes Service (AKS)
 
-Installation instructions for Gitpod Self-Hosted on Microsoft AKS are currently located in the [gitpod-io/gitpod-microsoft-aks-guide](https://github.com/gitpod-io/gitpod-microsoft-aks-guide) repository on GitHub. The installation process takes around forty minutes. In the end, the following resources are created:
+Instructions on how to greate an Microsoft AKS environment that is ready to install Gitpod are located in the [gitpod-io/gitpod-microsoft-aks-guide](https://github.com/gitpod-io/gitpod-microsoft-aks-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
```
Comment (MrSimonEmms) [inline gitpod/docs/self-hosted/latest/cluster-set-up/on-microsoft-aks.md:12]:
```suggestion
Instructions on how to create a Microsoft AKS environment that is ready to install Gitpod are located in the [gitpod-io/gitpod-microsoft-aks-guide](https://github.com/gitpod-io/gitpod-microsoft-aks-guide) repository on GitHub. The installation process takes around twenty minutes. In the end, the following resources are created:
```

**Review by MrSimonEmms — COMMENTED** [review]
Copy/pasta

**Comment by lucasvaltl** [comment]
Thanks for spotting that! Fixed :)

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**